src/rtk/features/overviewOrg/overviewOrgSlice.ts

```diff
@@ -0,0 +1,256 @@
+import axios from 'axios';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import { DeleteOrgDto } from '@/shared/dto/DeleteOrgDto';
+
+export interface OverviewListItem {
```

Comment:
I'll change it. I don't know what else to call OverviewOrg though...

Reply:
Hmm.. It can't just be Org, can it?
Or what were you aiming for with the Overview-part in the first place?

Reply:
we'll probably have a lot of redux state objects after a while. And we'll already get a new redux state object in the next feature we add which will be named DelegableOrg. Overview-part was to specify the context it is used in.

Reply:
Ooooooooh, well, in that case the name kinda makes sense.
Or, since were talking about orgs that already have things delegated to them, maybe DelegatedOrg?
It makes sense for what they are. The problem there I guess is that it looks very similar to the name DelegableOrg.. 🤔

Reply:
Yes I thought of that name too but I think it's a problem that it will be too similar to delegableOrg when you look in redux developer tools